src/services/s3.service.ts

```diff
@@ -11,7 +11,7 @@ aws.config.update({
 type SendData = aws.S3.ManagedUpload.SendData;
 const ManagedUpload = aws.S3.ManagedUpload;
 
-const bucket = S3_BUCKETNAME || '';
+const bucket = AWS_S3_BUCKETNAME || '';
```

Comment:
Are we checking what happens if the `AWS_S3_BUCKET` is undefined?
Is ok that the default value for this property is an empty string?

Reply:
you could fail fast in case `AWS_S3_BUCKETNAME` is missing while starting the app.

Reply:
Hey guys! 
We discussed this with Nico and decided to create a new issue to add a module to validate all the environment variables and their types, [similar to this implementation](https://stackoverflow.com/questions/61439809/environment-variables-validation-with-class-validator-in-nestjs).

Reply:
Ok, please make sure we create the issue!